Build Failure

**Describe the bug**
Build Failure

**To Reproduce**
Run the build script on the terminal.
 $ ./build.sh

**Log**
```shell
----------------------------------------
 Create Executable binary from GoMain
----------------------------------------
go build -a -ldflags '-extldflags "-static" -X main.version= -X main.commitID=cd0ed5b -X main.buildTime=20200706.1730 -X main.buildTags=' -o /home/t25kim/contribution/edge-home-orchestration-go/GoMain/bin/edge-orchestration main || exit 1
# controller/servicemgr/executor/containerexecutor
src/controller/servicemgr/executor/containerexecutor/ce_docker.go:49:10: undefined: docker
src/controller/servicemgr/executor/containerexecutor/ce_docker.go:53:17: undefined: docker
src/controller/servicemgr/executor/containerexecutor/ce_docker.go:78:2: not enough arguments to return
src/controller/servicemgr/executor/containerexecutor/ce_docker.go:88:2: not enough arguments to return
Makefile:65: recipe for target 'build-binary' failed
make: *** [build-binary] Error 1
```

**Test environment configuration (please complete the following information):**
* Firmware version: Ubuntu 16.04
* Hardware: x86-64
* Edge Orchestration Release: Baobab

--- a/src/controller/servicemgr/executor/containerexecutor/ce_docker.go
+++ b/src/controller/servicemgr/executor/containerexecutor/ce_docker.go
@@ -22,6 +22,7 @@ import (
 	"io"
 	"os"
 
+	"docker.io/go-docker"
 	"docker.io/go-docker/api/types"
 	"docker.io/go-docker/api/types/container"
 	"docker.io/go-docker/api/types/network"

